.pre-commit-config.yaml

```diff
@@ -21,11 +21,11 @@ repos:
       - id: end-of-file-fixer
       - id: trailing-whitespace
   - repo: https://github.com/psf/black-pre-commit-mirror
-    rev: 23.9.1
+    rev: 23.11.0
     hooks:
       - id: black
         language_version: python3
-        args: ['--target-version', 'py37']
+        args: ['--target-version', 'py311']
```

Comment:
as of 23.1.0, black can pick up the target version from pyproject.toml so this should no longer be needed (https://github.com/psf/black/releases/tag/23.1.0)

Reply:
Let's take care of this in #14.

Reply:
sounds like a plan

Reply:
Resolved in #28.